packages/web-pkg/src/composables/actions/files/useFileActionsCreateNewFile.ts

```diff
@@ -191,21 +184,15 @@ export const useFileActionsCreateNewFile = ({
 
     const inputSelectionRange = !areFileExtensionsShown.value
       ? null
-      : [0, defaultName.length - (extension.length + 1)]
+      : ([0, defaultName.length - (extension.length + 1)] as [number, number])
```

Comment:
maybe as const would work as well?

Reply:
As discussed this does not work the `const` type seems to be immutable.